CHANGES_UNRELEASED.md

```diff
@@ -18,3 +18,14 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
+
+## General
+### What's new
+- Uniffi was upgraded to 0.18.0. For our consumers, this means there now exported types that used to be internal to `uniffi`. ([#4949](https://github.com/mozilla/application-services/pull/4949)).
+  - The types are:
```

Comment:
Those are all of our `[Custom]` types so far (mostly in `places` and one in `nimbus`)